### ISSUE TITLE
a11y in dropdowns

### ISSUE THREAD

**Issue by robrez** [nik-christou/lit-element-bootstrap#9]
dropdowns seem to be missing accessibility features

https://getbootstrap.com/docs/4.3/components/dropdowns/#accessibility

**Comment by nik-christou** [comment]
Hi robrez,

I refactored dropdown and dropdown menu elements to include the same accessibility features as found in bootstrap. The following attributes were added:

- <bs-dropdown> has the aria-haspopup attribute
- <bs-dropdown-menu> has aria-expanded attribute
- <bs-dropdown-menu> has aria-labelledby attribute

Also the dropdown menu can be closed by pressing the ESC key and the menu items can be traversed via cursor keys (up and down).

you can find these changes either via the scoped package: @lit-element-bootstrap/dropdown@1.0.0-alpha.3 or the full package: lit-element-bootstrap@1.0.0-alpha.24
